File: common-client-libs/src/main/java/eu/getsoftware/hotelico/clients/infrastructure/exception/HotelException.java
package eu.getsoftware.hotelico.clients.infrastructure.exception;

/**
 *
 * <br/>
 * Created by e.fanshil
 * At 11.10.2018 16:40
 * 
 * Common Dashboard Exception
 */
public class HotelException extends Exception {
	
	String message;
	String reponseMessage;
		
	public HotelException(String message)
	{
		super(message);
		this.message = message;
		this.reponseMessage = message;
	}
}

File: common-client-libs/src/main/java/eu/getsoftware/hotelico/clients/infrastructure/exception/HotelException.kt
package eu.getsoftware.hotelico.clients.infrastructure.exception

/**
 *
 * <br/>
 * Created by e.fanshil
 * At 11.10.2018 16:40
 * 
 * Common Dashboard Exception
 */
class HotelException(message:String?) : Exception(message) {

    var reponseMessage: String? = message
}